executor/aggregate_test.go

```diff
@@ -269,7 +269,9 @@ func (s *testSuite) TestAggregation(c *C) {
 	result.Check(testkit.Rows("<nil>", "<nil>"))
 
 	result = tk.MustQuery("select count(*) from information_schema.columns")
-	result.Check(testkit.Rows("535"))
+	// When adding new memory table in information_schema, please update this variable.
```

Comment:
I mean can we count it dynamically, how about sum something like len(tableTriggersCols) ?